app/views/catalog/ds.html.erb

```diff
@@ -21,12 +22,7 @@
         # If the user is a repo admin or is permitted to manage both content and
         # rights, display raw xml editing
     %>
-      <%= if can?(:manage_item, @obj) &&
-             (
-               @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) ||
-               params[:dsid] == 'RELS-EXT'
-             ) &&
-             params[:dsid] != 'workflows'
+      <%= if can?(:manage_item, @obj) && Settings.editable_datastreams.include?(params[:dsid])
```

Comment:
the if should be after the "link_to" or make the if be in its own erb block. Then you can remove the rubocop disable stuff.